core/domain/stats_jobs_continuous_test.py

```diff
@@ -44,27 +43,19 @@ class MockInteractionAnswerSummariesAggregator(
     """A modified InteractionAnswerSummariesAggregator that does not start
     a new batch job when the previous one has finished.
     """
-    @classmethod
-    def _get_batch_job_manager_class(cls):
-        return MockInteractionAnswerSummariesMRJobManager
-
     @classmethod
     def _kickoff_batch_job_after_previous_one_ends(cls):
         pass
 
 
-class MockInteractionAnswerSummariesMRJobManager(
-        stats_jobs_continuous.InteractionAnswerSummariesMRJobManager):
-
-    @classmethod
-    def _get_continuous_computation_class(cls):
-        return MockInteractionAnswerSummariesAggregator
-
-
 class InteractionAnswerSummariesAggregatorTests(test_utils.GenericTestBase):
     """Tests for interaction answer view aggregations."""
 
-    ALL_CC_MANAGERS_FOR_TESTS = [MockInteractionAnswerSummariesAggregator]
+    def setUp(self):
+        super(InteractionAnswerSummariesAggregatorTests, self).setUp()
+        self.interaction_answer_summaries_aggregator_swap = self.swap(
```

Comment:
Be careful, this might trick users into thinking that swap is multi-use since you're using it many times. It'd be more readable to make this a method instead.

Reply:
Done